src/core/fonts.js

```diff
@@ -1074,6 +1075,21 @@ class Font {
     );
 
     fontName = stdFontMap[fontName] || nonStdFontMap[fontName] || fontName;
+
+    const stdFontMetrics = getFontBasicMetrics();
```

Comment:
Nit: Given that the name of the helper function was changed, maybe update this variable name accordingly?